ReactiveCocoa/Swift/SignalProducer.swift

```diff
@@ -1165,4 +1164,54 @@ extension SignalProducerType {
 	public func wait() -> Result<(), Error> {
 		return then(SignalProducer<(), Error>(value: ())).last() ?? .Success(())
 	}
+
+	/// Creates a new `SignalProducer` that will multicast values emitted by
```

Comment:
Maybe also add another paragraph discouraging the use of those operator unless it's strictly necessary, explaining it may introduce caching issues, etc?